Wikipedia/Code/ExploreViewController.swift

```diff
@@ -57,6 +57,9 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         super.viewWillAppear(animated)
         collectionViewUpdater?.isGranularUpdatingEnabled = true
         restoreScrollPositionIfNeeded()
+
+        /// Terrible hack to make back button text appropriate for iOS 14 - need to set the title on `WMFAppViewController`. For all app tabs, this is set in `viewDidAppear`.
+        parent?.navigationItem.backButtonTitle = title
```

Comment:
Yeah, I think this is the lowest-risk fix for this problem. It might also work if you embedded the 5 root view controllers in their own navigation controller, but I wouldn't be comfortable with a fix that big.

Reply:
Sounds good - I agree with you on that.